doc/classes/XRFaceTracker3D.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="XRFaceTracker3D" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A node for driving standard face meshes from [XRFaceTracker] weights.
+	</brief_description>
+	<description>
+		This node applies weights from a [XRFaceTracker] to a mesh with supporting face blend shapes.
+		The [url=https://docs.vrcft.io/docs/tutorial-avatars/tutorial-avatars-extras/unified-blendshapes]Unified Expressions[/url] blend shapes are supported, as well as ARKit and SRanipal blend shapes.
```

Comment:
If there's good links for the ARKIT and SRanipal ones it'd be nice as well

Reply:
The Unified Expressions website has a [Standard Comparison](https://docs.vrcft.io/docs/tutorial-avatars/tutorial-avatars-extras/compatibility/overview) page which explains the relationship between the Unified Expressions and the ARKit, SRanipal, and Meta standards.

Reply:
I'll add a link to the comparisons page to the documentation - hopefully their website doesn't change structure/paths too much.

Reply:
The link to the comparison with ARKit and SRanipal blend shapes is included.